### ISSUE TITLE
Clean task does not remove `libs` from project root

### ISSUE THREAD

**Issue by binkley** [cbeust/kobalt#175]
After using the `run` task, I ran the `clean` task but it did not remove the `libs` directory from project root.

This could be fixed, of course, though my preference is to not have a `libs` in project root in the first place -- see #173.

**Comment by cbeust** [comment]
`clean` only deletes what's under `kobaltBuild/` by design, why should it delete `$ROOT/libs`? That's dangerous, this directory could contain users libraries...

**Comment by binkley** [comment]
I noticed when I use `kobalt run` it creates a `libs` and copies my project jar there.  I have no user libraries separate from that created by Kobalt.  Do I misunderstand the design?

**Comment by cbeust** [comment]
No, it's just that `run` depends on `install`, which by definition, will
move the output of your build from `kobaltBuild/libs` to the `libs`
directory of your project...


-- 
Cédric

**Comment by cbeust** [comment]
No, it's just that `run` depends on `install` which moves the output of your build from `kobaltBuild/libs` to the `libs` directory of your project...
